job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/publisher/PublisherContext.groovy

```diff
@@ -1822,6 +1822,31 @@ class PublisherContext extends AbstractExtensibleContext {
         }
     }
 
+    /**
+     * Generates reports from static code violations detectors.
```

Comment:
ups... Thanks for that spot, I wanted to reuse some logic of another plugins in order to normalize the code and I did skip that comments. Thanks again